CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 
 # Unreleased
+ * Fixed exception message in AESCipher (thrown if key has unexpected length)
```

Comment:
I think we should categorize those into "Fixed", "Added", "Changed", "Removed", e.g. like we internally do with the fish filet library.